.github/workflows/reusable-build-docs.yaml

```
@@ -50,26 +47,34 @@ jobs:
           }}
         ref: ${{ inputs.repository-branch }}
         path: build-directory
+        persist-credentials: false
 
     - name: Setup nox
       uses: wntrblm/nox@2025.10.16
 
     - name: Output Python info
       run: python --version --version && which python
 
-    - name: Set the VERSION variable
-      run: echo VERSION="${PACKAGE_VERSION}" >> "${GITHUB_ENV}"
+    - name: Graft ansible-core
+      run: nox -s clone-core
+      working-directory: build-directory
 
     - name: Build the Ansible community package docs
-      run: >-
-        nox -e make -- webdocs ${{
+      env:
+        PACKAGE_VERSION: "${{ inputs.ansible-package-version }}"
+      run: |
+        # Clear PACKAGE_VERSION if it's set to devel
+        if [ "${PACKAGE_VERSION}" = "devel" ]; then
+          PACKAGE_VERSION=""
+        fi
```

We can move the conditional here instead of bash
```suggestion
        PACKAGE_VERSION: >-
          ${{
            inputs.ansible-package-version! = 'devel'
            && inputs.ansible-package-version
            || ''
          }}
      run: >-
```

---

It should likely be ` != 'devel'`, not `! = 'devel'` :)

---

Corrected, thanks. Sometimes I make these suggestions from mobile and miss when it automatically moves whitespaces..